src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="/" 
+      method="post" 
+      class="bigForm" 
```

Comment:
no camelCase for class names, only kebab-case. fix everywhere

Reply:
ok